src/utils/isProduction.ts

```diff
@@ -1,3 +1,5 @@
 export default function isProduction() {
-  return typeof window !== 'undefined' && window.location.origin === 'https://gallery.so';
+  return (
+    process.env.NEXT_PUBLIC_VERCEL_ENV === 'production' || process.env.NODE_ENV === 'production'
```

Comment:
hmm doesn't a vercel deploy mark `NODE_ENV` as `production`? even if it's to dev?

Reply:
at build time

Reply:
you're right! fixed https://github.com/gallery-so/gallery/pull/570/commits/df6763520534ed00628d6623952b45c14c2cff03